src/main/java/com/github/manolo8/darkbot/core/utils/Lazy.java

```diff
@@ -6,41 +6,52 @@
 import java.util.function.Consumer;
 
 public class Lazy<C> {
+    protected final List<Consumer<C>> consumers = new ArrayList<>();
+    protected C value;
```

Comment:
Keep this public for a bit, make sure that the plugins can be updated to use `get()` if any of them use this